lib/tapioca/compilers/dsl/active_support_current_attributes.rb

```diff
@@ -0,0 +1,120 @@
+# typed: true
+# frozen_string_literal: true
+
+require "parlour"
+
```

Comment:
I didn't have to `require "active_support"` here and therefore did not add a `rescue` block.

Is it possible that loading this file on systems without `ActiveSupport` will cause an error?

Reply:
Yes, it probably would. Let's add the explicit `require "active_support"` here and the `rescue` block for it.